### PR TITLE
Update Channel Selector initial (1.0.2) by Jose Manuel Delicado)

### PR DESCRIPTION
**Pull request by josephsl**
Hi,

This add-on was reviewed in 2019 and now being registered for community distribution (ap9ologies for this lateness)...

### Introduction:

This add-on allows users to select update chanel for NVDA screen reader without having to visit snapsthos page.

## Release information:

* Name: Update Channel Selector
* Author: jose Manuel Delicado
* Repo: https://github.com/nvda-es/updateChannel
* Version: 1.0.2
* Add-on update channel: stable
* NVDA compatibility: 2019.1 to 2020.4
* Add-on files update key: updchannelselect
* Basic review reference thread: https://nvda-addons.groups.io/g/nvda-addons/topic/34396512#10965
* Basic review: license and copyright/pass, documentation/pass, security/pass, user experience/pass

Thanks.

### CODE REVIEW

**Comment by jmdaweb** [comment]
Hi,
All critical user experience issues mentioned in [this thread](https://nvda-addons.groups.io/g/nvda-addons/topic/82924382) have been fixed. The download link keeps unchanged. Again, thank you very much for your feedback, @nvdaes.
Regards.

**Review by nvdaes — APPROVED** [review]

**Comment by nvdaes** [comment]
@feerrenrut, this has been reviewed again.


Important: please consider the following issues before merging this PR:


- https://github.com/nvaccess/nvda/issues/6149
- https://github.com/nvaccess/nvda/issues/10679
Also, note that, for example, if we select the beta channel, the add-on can offert this version: beta-18478,0887269f 

See also this message, reporting that an old stable version is offered: 
https://nvda-addons.groups.io/g/nvda-addons/message/15146

I'm approving this based on user experience related to gui, but you should decide what to do since other issues are related to interaction with NV Access server.

sha256: 7e6256a286cfb25b098a4afd4d965b6d8c4d8a946c2e7fad4af8d6d360b82964

**Comment by feerrenrut** [comment]
I'm going to accept this change. I don't think we should dictate the UX standards for end users, the trade-offs made by the developer may be acceptable to many. Most "app stores" get around this by allowing users to leave a review.

**Comment by nvdaes** [comment]
OK, received.

**Pull request merged**
